gradle/verification-metadata.xml

```diff
@@ -1111,6 +1109,11 @@
             <pgp value="9e93db1192cee3d3b581aabb37e9d58ea4598641"/>
          </artifact>
       </component>
+      <component group="org.javassist" name="javassist" version="3.24.0-GA">
+         <artifact name="javassist-3.24.0-GA.jar">
+            <pgp value="666a4692ce11b7b3f4eb7b3410066a9707090cf9"/>
+         </artifact>
+      </component>
```

Comment:
Why is this needed?

Reply:
That would be a question for @melix .... Should I check in this change?

Reply:
You should tell me ;) If it works _without_ you shouldn't. If it works with it, you should check the signature and add it.